src/abstractWrappers/Disputes.js

```diff
@@ -332,7 +333,8 @@ class Disputes extends AbstractWrapper {
           this.getDataForDispute(
             dispute.arbitratorAddress,
             dispute.disputeId,
-            account
+            account,
+            arbitratorData
```

Comment:
I just noticed its arbitatorData not arbitratorExtraData. Still shouldn't need to pass it

Reply:
It would take way too long to fetch that for every dispute and doesn't make sense if you are fetching a list of them.

Reply:
At the very least we should split them up into different and explicit params as arbitratorData can change. I am not sure I agree with you though. It is only a marginal cost to look up the session and period on the fly and there is no reason why we should have to pass it since you will only ever be passing the current session and period. It doesn't give the user any extra options, just extra work and it's something we can easily get from the current params. Less params are always better

Reply:
@satello 
How can you get `canRepartition` from the current params?

Reply:
Ideally, I think `KlerosWrapper` should cache session and period, and change it through events. That way we just access the property.

Reply:
By using arbitratorAddress + disputeId to look it up. Caching is potentially a good idea but it could cause tx's to fail since events aren't necessarily real time so they are more unreliable than polling

Reply:
Yes, but you would need to use `KlerosWrapper.getData` for every dispute in the list. That would be crazily inefficient.

Reply:
I'll change it for now and we can figure out the cache thing later.